### PR TITLE
Logging assert watchlist

### LINKED ISSUE #85: Add logging and assertions for Watchlist

### PR DESCRIPTION
**Pull request by wwweert123**
Fix #85 Please take a look guys!

### CODE REVIEW

**Review by ryan1604 — APPROVED** [review]
Looks good bro! I like the clear error messages. Maybe capitalise the first letter of the message in line 54 of Stock.java to standardise

**Review by NeoMinWei — APPROVED** [review]
LGTM
Take note there is commented out log in Watchlist class, can either uncomment it or remove

File: src/main/java/seedu/financialplanner/investments/WatchList.java
Comment (NeoMinWei) [inline src/main/java/seedu/financialplanner/investments/WatchList.java:1]: thanks for removing commented out code, makes the overall code neater

**Review by NeoMinWei — COMMENTED** [review]
LGTM

**Pull request merged**